### DISCUSSION TITLE
[@types/node] The question about node dependency undici-type version

### DISCUSSION THREAD

**Discussion by GitHubProfessor** [DefinitelyTyped/DefinitelyTyped#68776] [Issues with a @types package]
@types/node is denpending on undici-type:5.26.4 now.
 but, black duck checked hight risk about undici-type:5.26.4,it requires to use the version of undici-type:5.28.3 or undici-type:6.6.2. 
when @types/node can update the undici-type version? thanks!!

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Undici was updated to v5.28.3 in Node.js v20.11.1
https://github.com/nodejs/node/releases/tag/v20.11.1
So undici-type can be updated to 5.28.3

**Comment by Semigradsky** [comment]
But I think it can't be any high risk about undici-type because it is a package with types only.
